walkthroughs/week-2-web-development/examples/stanley/src/main/webapp/style.css

```diff
@@ -1,25 +1,115 @@
-#content {
-  margin-left: auto;
-  margin-right: auto;
-  width: 650px;
-}
+@media screen and (min-width: 600px)
+{
+  .header-container
+  {
+    display: flex;
+    align-items: center;
+    justify-content: space-evenly;
+    height: 100px;
+    width:100%;
```

Comment:
Please add a space after the colon here and below.

Reply:
done